pkg/testrunner/scenario.go

```diff
@@ -74,7 +73,7 @@ type Generation struct {
 // it may not work as expected.
 func RootDir() string {
 	_, b, _, _ := runtime.Caller(0)
-	d := path.Join(path.Dir(b))
+	d := ospath.Join(ospath.Dir(b))
```

Comment:
@afzal442 could we import this as path instead?

Reply:
If I do so, I am getting error for other method as below

```error
path.Join undefined (type string has no field or method Join)
```